tests/proc.rs

```diff
@@ -1,12 +1,27 @@
 #[test]
-fn test_pio_proc() {
-    let p = pio_proc::pio!(
-        1,
-        "
-    label:
-        jmp label
-    "
+fn test_file() {
+    let p = pio_proc::pio_file!(
+        "../tests/test.pio",
+        select_program("test2"),
+        options(max_program_size = 32)
     );
+    assert_eq!(p.program.origin, Some(5));
+    assert_eq!(&*p.program.code, &[0, 0]);
+    assert_eq!(
+        p.program.wrap,
+        pio::Wrap {
+            source: 0,
+            target: 0
+        }
+    );
+    assert_eq!(p.public_defines.label, 0);
+    assert_eq!(p.public_defines.owo, 2);
+}
+// }
```

Comment:
?

Reply:
Should I fix this?

Reply:
I'm assuming we're referring to the commented closing brace here?
Please remove it, thanks!